nebula/ui/components/inAppAuth/MZInAppAuthenticationBase.qml

```diff
@@ -59,6 +60,8 @@ MZFlickable {
                 Layout.preferredHeight: MZTheme.theme.rowHeight
                 Layout.preferredWidth: MZTheme.theme.rowHeight
                 Layout.leftMargin: MZTheme.theme.windowMargin / 2
+                enabled: _backButtonVisible
```

Comment:
any reason we need to disable AND hide vs just hide?

Reply:
We absolutely do not, good catch. Thanks, updated.